src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -13489,6 +13489,14 @@ namespace AirflowNetwork {
         }
     }
 
+    bool Solver::isLoopOn(int airLoopNum) const
+    {
+        if ((airLoopNum <= 0) || (airLoopNum > LoopOnOffFlag.isize())) {
+            return false;
+        }
+        return this->LoopOnOffFlag(airLoopNum);
+    }
```

Comment:
safe getter definition, simple.

Reply:
No offense, but this is a band-aid. Actually fixing this requires figuring out why this array was given a bad index to start with and fixing that problem. The accretion of these types of band-aids is one of the reasons why we had to do an EnergyPlus 10X project.

Reply:
There are 18 AirflowNetwork:Distribution:Component:Fan objects and 18 AirloopHVAC objects in the defect file. If all of these fans are used in air loops, which they should be, then there is an issue with setting state.dataAirflowNetwork->DisSysCompCVFData(i).AirLoopNum for one or more of these fans.

Reply:
Nope, AirLoopNum is set in function AssignFanAirLoopNum called from ManageAirflowNetworkBalance. Seems like a calling order issue.

Reply:
@rraustad I think you need to pull in develop into whatever branch you're looking at.

Reply:
https://github.com/NREL/EnergyPlus/issues/9286#issue-1143897256

> In the model, the attic isn't associated with any air loop (but is included in the AFN for infiltration calculation purposes) so AirLoopNum = 0.

Why would AirLoopNum not be 0 in that case? Or is that disallowed somehow?

Rest assured that I didn't mean to introduce a band aid though. E+ does have a lot of structs with integers Num to access optional element/configurations and if num = 0, then that optional component isn't there. And this is a complex area of the E+ code that I don't claim to understand.

Maybe I did go down the wrong path here because I misdiagnosed the issue.
Perhaps it would be best if I just closed the PR and let someone more familiar with the AFN fix it.